test/unit/db/repos/delegates.js

```diff
@@ -165,10 +167,50 @@ describe('db', () => {
 				return expect(db.one.firstCall.args[1]).to.eql([]);
 			});
 
-			// eslint-disable-next-line mocha/no-skipped-tests
-			it.skip('should return list of duplicate delegates', done => {
-				// TODO: Dependent on PR https://github.com/LiskHQ/lisk/pull/1607
-				done();
+			it('should return list of duplicate delegates', function*() {
+				const block = seeder.getLastBlock();
+
+				const trs1 = transactionsFixtures.Transaction({
+					blockId: block.id,
+					type: 2,
+				});
+				yield db.transactions.save(trs1);
+				yield db.query(
+					db.$config.pgp.helpers.insert(
+						{
+							transactionId: trs1.id,
+							username: randomstring.generate({
+								length: 10,
+								charset: 'alphabetic',
+							}),
```

Comment:
This kind of randomness is probably not too dangerous (at least length is fixed) but does it doesn't bring any value - If we get an error, how would we reproduce; if it fails randomly, we won't know which specific string caused the error.

Reply:
@jondubois As per my understandings randomizing input gave you variety of input on each run. As we run tests many times to finally reached to release, so this way we can test different variety of input values. And its not here, we are using this approach across all test suit.